test/test_package.cpp

```diff
@@ -1436,3 +1436,103 @@ TEST(DDPackageTest, expectationValueExceptions) {
 
     EXPECT_ANY_THROW(dd->expectationValue(xGate, zeroState));
 }
+
+TEST(DDPackageTest, DDFromSingleQubitMatrix) {
+    auto row0        = dd::CVec{0, 1};
+    auto row1        = dd::CVec{2, 3};
+    auto inputMatrix = dd::CMat{row0, row1};
```

Comment:
```suggestion
    const auto inputMatrix = dd::CMat{{0, 1}, {2, 3}};
```
can be simplified. Note that this is not a valid unitary matrix and, thus, probably not the best test example here. Why not use some known quantum operation matrix? 


Furthermore, it can be `const`.

Reply:
Done

Extra question: Is there a limitation with non-unitary matrices?

Reply:
No, not at all. 
However, I'd guess that the implementation is particularly optimized for unitary matrices.
The normalization scheme that is employed ensures that the real and imaginary part of any number in the DD are in the intervall `[0, 1]` and the ComplexTable is especially optimized around that assumption holding throughout every interaction with the package.
So, if a non-unitary matrix might result in edge weights not in the unit interval, the implementation will still work, but it won't be efficient any more, as all values larger than 1.0 will be put into the same hash bucket in the ComplexTable.